library/common/network/BUILD

```diff
@@ -76,3 +76,23 @@ envoy_cc_library(
         "@envoy//source/common/network:socket_interface_lib",
     ],
 )
+
+cc_library(
+    name = "ios_platform_verifier",
+    srcs = select({
```

Comment:
Any reason to put these selects in multiple places turning this into an empty `cc_library` vs using a single `select` when this is used in `deps`?

Reply:
Ah I didn't think about keeping `//library/common/network/...` compiling on every platform, that's a good enough reason to do this I guess.

Reply:
Oh, interesting. That's basically the only way I build stuff :) FWIW, Envoy has `envoy_cc_linux_library`, `envoy_cc_posix_library`, `envoy_cc_win32_library` which wrap these select statements. We could use this in Envoy Mobile, if we wished?

Reply:
An empty library seems fine for now.